MANUAL.txt

```diff
@@ -7277,6 +7277,9 @@ definitions](https://github.com/KDE/syntax-highlighting/tree/master/data/syntax)
 
 To disable highlighting, use the `--no-highlight` option.
 
+If you receive an error that pandoc `Could not read highlighting theme`,
+check that the JSON file is encoded with UTF-8 and has no Byte-Order Mark (BOM).
+
```

Comment:
This paragraph may no longer be necessary (given the change to the example), but if it is kept it should go before the previous paragraph, because it concerns `--syntax-definition`.   Also I'd suggest using double quotes:
```
"Could not read highlighting theme"
```

Are you sure that a BOM causes a problem?  That would surprise me; generally we have set things up so that a BOM is ignored, and if there's somewhere it isn't, we might want to take a look at that.

Reply:
I have tested and BOM is still an issue in a theme file.

You mentioned it in this 2021 comment: https://github.com/jgm/pandoc/issues/6704#issuecomment-832075056

Paragraph order changed

Reply:
Oh yes, I see now "the BOM is illegal in JSON."  So, this comment is worth making.